File: Giggy-Server/graph/model/session.go
package model

import (
	"github.com/google/uuid"
)

type Signin struct {
	UserID           uuid.UUID `json:"user_id"`
	Token            string    `json:"token"`
	HasFarmingRights bool      `json:"has_farming_rights"`
	HasPosterRights  bool      `json:"has_poster_rights"`
}

File: Android/app/src/main/java/com/lomolo/copod/model/Session.kt
package com.lomolo.copod.model

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity("sessions")
data class Session(
    @PrimaryKey val id: String = "",
    var token: String = "",
    var hasFarmingRights: Boolean = false,
    var hasPosterRights: Boolean = false,
)